src/object/schema_manager.c

```diff
@@ -3312,10 +3321,22 @@ sm_check_system_class_by_name (const char *name)
       return false;
     }
 
-  count = sizeof (system_classes) / sizeof (system_classes[0]);
-  for (i = 0; i < count; i++)
+  /* binary search */
```

Comment:
How about use bsearch()
 ```
      #include <stdlib.h>

       void *bsearch(const void *key, const void *base,
                     size_t nmemb, size_t size,
                     int (*compar)(const void *, const void *));
```

Reply:
Because of the small amount of data, I decided not to use binary search.
Instead, it was improved as a way to compare lengths.